org.eclipse.winery.repository.ui/src/app/wineryEditorModule/editor-builder.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
```

Comment:
I think we should create a `const orion` here because of https://travis-ci.org/OpenTOSCA/winery/jobs/392301275#L6787